tools/webots.py

```diff
@@ -178,9 +198,9 @@ def exec_push():
         sys.exit(exit_code)
 
 
-def run(sub_command, roles=None, role=None, **kwargs):
+def run(sub_command, roles=None, role=None, target="generic", **kwargs):
     if sub_command == "build":
-        exec_build(roles)
+        exec_build(target, roles)
     elif sub_command == "push":
         exec_push()
     elif sub_command == "run":  # For testing
```

Comment:
What is this comment? Not very clear what you are trying to do here

Reply:
The image built is intended to be ran on AWS for RoboCup, the `./b webots run` is usually used just for testing the built image.